### ISSUE TITLE
Weird JAWS bug on User Details page

### ISSUE THREAD

**Issue by weboverhauls** [ginader/easychirp#163]
From user:

The issue occurs in Jaws on the User Details screen in where I cannot tab
to/operate the  "follow" link (happens in both FF and IE) as expected.
I can nav to it with the arrow keys, and it does show up in the list
of links view. However, if I nav to it with the arrows and attempt to
activate the link with Enter, nothing happens. I can only activate the
link if I bring up a list of links, navigate to the "follow" link from
there,  and then use the Activate Link feature in there.
